includes/classes/PluginManager.php

```diff
@@ -28,7 +28,7 @@ public function inspectAndUpdate()
 
     public function getInstalledPlugins()
     {
-        $results = $this->pluginControl->where(['status' => 1])->get();
+        $results = $this->pluginControl->where(['status' => 1])->orderBy('unique_key')->get();
```

Comment:
Do we want it to be sorted "first" by `unique_key`, which might be very different than alpha-by-name?

Or would `->orderBy('name')->orderBy('unique_key')` (first alpha by name, then by key for duplicate names) be better?

Reply:
Done; see commit below.